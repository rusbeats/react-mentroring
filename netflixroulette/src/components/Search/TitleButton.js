/* eslint-disable import/no-extraneous-dependencies */
import React from 'react';
import styled from 'styled-components';

// Homework 8: chose Styled Components
const ButtonTitle = styled.button`
  color:white;
  background-color:#fb6e6e;
  font-size:25px;
  text-transform: uppercase;
`;
const TitleButton = () => (
  <ButtonTitle>title</ButtonTitle>
);
export default TitleButton;
