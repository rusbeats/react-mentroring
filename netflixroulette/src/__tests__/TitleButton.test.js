import React from 'react';
import { shallow } from 'enzyme';
import TitleButton from '../components/Search/TitleButton';

// Homework 4: Enzyme Snapshot testing
it('TitleButton renders correctly', () => {
  const component = shallow(
    <TitleButton />
  );
  expect(component).toMatchSnapshot();
});
