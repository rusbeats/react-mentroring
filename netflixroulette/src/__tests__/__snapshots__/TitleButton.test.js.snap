// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleButton renders correctly 1`] = `
<styled.button>
  title
</styled.button>
`;
